web/src/components/Incidents/AlertsChart/AlertsChart.tsx

```diff
@@ -30,6 +30,7 @@ import { useTranslation } from 'react-i18next';
 import { AlertsChartBar } from '../model';
 import { setAlertsAreLoading } from '../../../store/actions';
 import { MonitoringState } from '../../../store/store';
+import * as _ from 'lodash-es';
```

Comment:
we should avoid importing the whole library and use only the functions needed. Even if this already present on the code we are moving away from it

Reply:
Replaced it with only isEmpty, force pushed the update